README.md

```diff
@@ -10,7 +10,7 @@ Issues and tasks for all three of those repositories are tracked here. We have a
 
 We welcome contributions to help us improve and complete the .NET docs. To contribute, see the [Projects for .NET Community Contributors](https://github.com/dotnet/docs/projects/35) for ideas. The [Contributing Guide](CONTRIBUTING.md) has instructions on procedures we use. Or, check the [issues list](https://github.com/dotnet/docs/issues) for tasks that interest you.
 
-We anticipate that [Xamarin](https://docs.microsoft.com/xamarin), [Mono](http://docs.go-mono.com/?link=root%3a%2fclasslib) and [Unity](http://docs.unity3d.com/Manual/index.html) will also use this documentation.
+We anticipate that [Xamarin](https://docs.microsoft.com/xamarin), [Mono](https://docs.go-mono.com/?link=root%3a%2fclasslib) and [Unity](https://docs.unity3d.com/Manual/index.html) will also use this documentation.
```

Comment:
The Mono link is broken; the correct link is <https://www.mono-project.com/docs/>.

Reply:
The original `http` link is OK. This replacement was caused by a bug. I have reverted it.